test/test_helper.rb

```diff
@@ -79,9 +80,9 @@ def require_test_classes
   end
 
   fixtures = if ActiveRecord::VERSION::MAJOR >= 4
-               RAILS_5_FIXTURE_CLASSES + BASE_FIXTURE_CLASSES
+               RAILS_5_FIXTURE_CLASSES
```

Comment:
what's with this change?

Reply:
`BASE_FIXTURE_CLASSES` are added just after `if` block, I removed duplication. This change is not necessity for PR, just a little code cleaning. Should I revert it?

Reply:
Nah, I just didn't notice the duplication. 👍